doc/release/1.13.0-notes.rst

```diff
@@ -184,6 +184,20 @@ of arrays.
 
 It is similar to Matlab's square bracket notation for creating block matrices.
 
+Numpy may be built with relaxed stride checking debugging
+---------------------------------------------------------
+Setting NPY_RELAXED_STRIDES_DEBUG=1 in the enviroment when relaxed stride
+checking is enabled will cause numpy to be compiled with the affected strides
+set to the maximum value of npy_intp in order to help detect invalid usage of
```

Comment:
Oh, I agree it's not necessary (thanks for the formal explanation though) - but am I right in saying it would be valid?